### DISCUSSION TITLE
php 7.3 started but not executing?

### DISCUSSION THREAD

**Discussion by sjmike** [Homebrew/discussions#1618] [Getting started]
I'm on Mac, latest OS. Working to try and get Magento running. I show the following:
```
`brew services list

Name    Status  User       Plist
httpd   error   MyUser /Users/mikelatzky/Library/LaunchAgents/homebrew.mxcl.httpd.plist
mysql   started MyUser /Users/mikelatzky/Library/LaunchAgents/homebrew.mxcl.mysql.plist
php@7.3 started MyUser /Users/mikelatzky/Library/LaunchAgents/homebrew.mxcl.php@7.3.plist`

```
1. Not sure why I show error for hatted, but it seems to need 'sudo apachectl restart' any time I make a change. 'brew services restart httpd' doesn't cause a change in my browser/configuration....
2. php shows it's running (above), but when I execute phpversion(); in a .php file from the browser like this (filename phpinfo.php) by going to localhost/phpinfo.php:
`<?php
echo 'PHP version: ' . phpversion();
`
 I get:
`<?php
echo 'PHP version: ' . phpversion();`

in my browser..... not the execution/php information I should expect to see. I've checked this in Safari and Chrome....

Any ideas? 

TIA

**Comment by SMillerDev** [comment]
> sudo apachectl restart

This runs it as root. So `brew services` can't run on the same port.

**Reply by sjmike** [reply]
That makes sense, but several sites giving step by step instruction to set up Magento under Homebrew on the Mac are all saying to switch the code to use port 80. 

The server works, but php doesn't seem to be...

One example is:
`https://alpineinc.com/2020/05/install-magento-2-locally-on-a-mac/`

Not sure where to go from here - I'd seem to be ok if I could get php to respond properly....

**Reply by SMillerDev** [reply]
Contact the writer of the blog. I can't help them writing shitty instructions
